pipelinewise/cli/config.py

```diff
@@ -318,6 +319,13 @@ def save_tap_jsons(self, target, tap, extra_config_keys=None):
         # Generate tap inheritable_config dict
         tap_inheritable_config = utils.delete_empty_keys({
             'temp_dir': self.get_temp_dir(),
+            'tap_id': tap.get('id'),
+            'query_tag': json.dumps({
+                'ppw_component': tap.get('type'),
+                'tap_id': tap.get('id'),
+                'schema': '{schema}',
```

Comment:
Is this supposed to be a f-string?

Reply:
Asking because `'{schema}'` will output `'{schema}'`.

Reply:
same with `table` below

Reply:
Oh, I think this is meant to be a template or a placeholder, right?

Reply:
yes, `query_tag` is a templated string where `{schema}` and `{table}` are tokens replaced dynamically at runtime by target-snowflake.

Reply:
Ok.

Reply:
I'm a bit worried here, so in this [function](https://github.com/transferwise/pipelinewise/blob/25c13bbd0adbd0c3cb76dfd2b6a45e069c61005b/pipelinewise/cli/tap_properties.py#L93) in PPW, we use double curly brackets for templated strings `{{ token }}`, but here it's a single one.
I'm afraid we're not being consistent here!

Reply:
Single brackets used in configurable string templates in target-s3-csv (`naming_convention` parameter) and in target-snowflake (`query_tag` parameter). So PPW targets components in sync.
 
Double brackets used in PPW in the above function to generate and not configurable.

What's the reason of the worries, what issues it can cause?
And which pattern we should use, double or single brackets? I'm afraid both cases requires changes in multiple places in multiple repositories ånd I think is out of the scope of this PR.

Reply:
hmm, we don't want to beak anything.

I guess we can leave this for another PR.

Reply:
no worries, I'll update the PRs to use jinja style double brackets for query tagging. Double brackets sounds like the correct option we should go with.

target-s3-csv will remain the same, and will use single brackets. We'll address that separately whenever it's needed. What do you think, is that a good plan?